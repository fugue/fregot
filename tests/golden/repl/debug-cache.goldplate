{
    "command": "fregot",
    "arguments": ["repl", "--no-history-file"],
    "asserts": [
        {"exit_code": 0},
        {"stderr": "${GOLDPLATE_NAME}.stderr"},
        {"stdout": "${GOLDPLATE_NAME}.stdout"}
    ],
    "stdin": [
        ":load debug-cache.rego",
        "test_allow",
        ":break test_allow",
        "test_allow",
        ":continue"
    ]
}
