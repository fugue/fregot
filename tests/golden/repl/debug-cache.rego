package fregot.tests.repl.debug_cache

allow {
    input.user == "alice"
} {
    input.user == "bob"
}

test_allow {
    allow with input as {"user": "bob"}
}
