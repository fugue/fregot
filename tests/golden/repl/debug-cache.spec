{
    "command": "fregot",
    "arguments": ["repl"],
    "asserts": [
        {"exit_code": 0},
        {"stderr": "${SPEC_NAME}.stderr"},
        {"stdout": "${SPEC_NAME}.stdout"}
    ],
    "stdin": [
        ":load debug-cache.rego",
        "test_allow",
        ":break test_allow",
        "test_allow",
        ":continue"
    ]
}
