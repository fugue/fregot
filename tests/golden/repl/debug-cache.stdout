repl% fregot.tests.repl.debug_cache% = true
fregot.tests.repl.debug_cache% fregot.tests.repl.debug_cache% 10|     allow with input as {"user": "bob"}
        ^^^^^
fregot.tests.repl.debug_cache(debug)% (debug) = true
(debug) finished
fregot.tests.repl.debug_cache% 